<?xml version="1.0" encoding="UTF-8"?>
<launch>
  <!-- Arguments -->
  <arg name="initial_ts_state_from_agent" default="False"/>
<!--   <arg name="num_mobile" default=1/> -->
<!--   <arg name="num_quadruped" default=1/> -->

  <arg name="agent_name_mobile_1" default="dr_0"/>
  <arg name="agent_name_mobile_2" default="wassi_0"/>
  <arg name="agent_name_dog_1" default="a1_gazebo"/>
  <arg name="rviz" default="true"/>

  <include file="$(find ltl_automation_a1)/launch/dr_monitor_bt.launch">
    <arg name="agent_name" value="$(arg agent_name_mobile_1)" />
    <arg name="plan_name" value="/action_plan_1" />
    <arg name="pose_x" value="-10" />
    <arg name="pose_y" value="21" />
    <arg name="init_yaw" value="3.14" />
  </include>

  <include file="$(find ltl_automation_a1)/launch/wassi_monitor_bt.launch">
    <arg name="agent_name" value="$(arg agent_name_mobile_2)" />
    <arg name="plan_name" value="/action_plan_3" />
    <arg name="pose_x" value="7.0" />
    <arg name="pose_y" value="6.0" />
    <arg name="init_yaw" value="3.14" />
  </include>

  <include file="$(find ltl_automation_a1)/launch/a1_monitor_bt.launch">
    <arg name="agent_name" value="$(arg agent_name_dog_1)" />
    <arg name="plan_name" value="/action_plan_2" />
    <arg name="pose_x" value="-16.58" />
    <arg name="pose_y" value="12.0" />
    <arg name="init_yaw" value="3.14" />
  </include>


  <node if="$(arg rviz)" name="rviz" pkg="rviz" type="rviz"
        args="-d $(find unitree_navigate)/rviz/multi_robot_mocap.rviz -f map"
        output="screen"/>


</launch>
